README.md

```diff
@@ -32,6 +32,62 @@ YAS is a pet project aim to practice building a typical microservice application
 1. Open another browser tab and go to http://identity.yas.local. Admin account is: admin/admin
 1. The Postgresql server is also published to the host machine: servername: localhost, port: 30007, username: admin, password: admin
 
+## Getting started with Docker in Ubuntu
+
+1. Get the latest source code
+2. Install minikube
+
+    ```sh
+    curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+    sudo install minikube-linux-amd64 /usr/local/bin/minikube 
+    ```
+3. Create folders 
+    ```sh
+    mkdir -p -v $HOME/k8s-yas-data/postgres \
+                $HOME/k8s-yas-data/pgadmin \
+                $HOME/k8s-yas-data/var/lib/postgres \
+                $HOME/k8s-yas-data/var/lib/pgadmin
+    ```
+4. Start minikube 
+    ```sh
+    minikube start --driver=docker --mount-string $HOME/k8s-yas-data:/k8s-yas-data/ --mount 
+   ```
+5. Install kubectl 
+    ```sh
+    minikube kubectl -- get pods -A
+    echo 'alias kubectl="minikube kubectl --"' >> ~/.bashrc
+   source ~/.bashrc
+   ```
+6. Install NGINX ingress controller 
+    ```sh
```

Comment:
This command is for Docker Desktop for Minikube we should run `minikube addons enable ingress` ?